app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Recipes</h1>
+<% @recipes.each do |recipe| %>
+<a href="recipes/<%= recipe.id %>">Recipe #<%= recipe.id %></a>
+<div>Name: <%= recipe.name %> by <%= recipe.user.name %></div>
+<div>Description: <%= recipe.description %></div>
```

Comment:
- Kindly add the total food items and price following the wireframe. It's a project requirement.

![image](https://user-images.githubusercontent.com/47673311/193906388-7a8fe754-9086-4dd7-b9a4-bef865d8756a.png)